### PR TITLE
Update packages for security

### PR DESCRIPTION
**Pull request by dmannarino**
## Pull request checklist

Please check if your PR fulfills the following requirements:
- [x] Make sure you are requesting to pull a topic/feature/bugfix branch (right side). Don't request your master!
- [x] Make sure you are making a pull request against the develop branch (left side). Also you should start your branch off our develop.
- [x] Check the commit's or even all commits' message styles matches our requested structure.
- [x] Check your code additions will fail neither code linting checks nor unit test.

## Pull request type

 

 

Please check the type of change your PR introduces:
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [x] Other (please describe): 
Package update (for security)

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?

-
-
-

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Review by danscales — COMMENTED** [review]
Seems fine.   I assume all the package updates for security are hidden in the changes to Pipfile.lock?  If there were any particular packages that you updated that were the most important to update for security issues, might be interesting to mention them with their new versions in a description of a new commit (dummy commit, if there are no other requested changes).  But don't worry about it if that's a pain.

**Comment by dmannarino** [comment]
> Seems fine. I assume all the package updates for security are hidden in the changes to Pipfile.lock? If there were any particular packages that you updated that were the most important to update for security issues, might be interesting to mention them with their new versions in a description of a new commit (dummy commit, if there are no other requested changes). But don't worry about it if that's a pain.

Good idea, will do!

**Comment by dmannarino** [comment]
> Seems fine. I assume all the package updates for security are hidden in the changes to Pipfile.lock? If there were any particular packages that you updated that were the most important to update for security issues, might be interesting to mention them with their new versions in a description of a new commit (dummy commit, if there are no other requested changes). But don't worry about it if that's a pain.

Done in this commit: https://github.com/wri/gfw-data-api/pull/416/commits/cd2a7757af1e3b6b750d44bbf7049703f9db82cd

File: tests_v2/unit/app/routes/datasets/test_query.py [hunk 1]
```diff
@@ -235,11 +236,10 @@ async def test_redirect_get_query(
         params=params,
         follow_redirects=False,
     )
-
     assert response.status_code == 308
     assert (
-        response.headers["location"]
-        == f"/dataset/{dataset_name}/{version_name}/query/json?{response.request.url.query.decode('utf-8')}"
+        parse_qsl(urlparse(response.headers["location"]).query, strict_parsing=True)
+        == parse_qsl(urlparse(f"/dataset/{dataset_name}/{version_name}/query/json?{response.request.url.query.decode('utf-8')}").query, strict_parsing=True)
```
Comment (gtempus) [inline tests_v2/unit/app/routes/datasets/test_query.py:242]: I like it! 💪

**Review by gtempus — APPROVED** [review]
I like the test and project updates! But this PR title is a little misleading.... 😏

**Pull request merged**